angular-workspace/projects/ni/nimble-angular/package.json

```diff
@@ -26,9 +26,15 @@
     "@angular/common": "^14.2.0",
     "@angular/core": "^14.2.0",
     "@angular/forms": "^14.2.0",
+    "@angular/localize": "^14.2.0",
     "@angular/router": "^14.2.0",
     "@ni/nimble-components": "^19.8.2"
   },
+  "peerDependenciesMeta": {
+    "@angular/localize": {
+      "optional": true
+    }
+  },
```

Comment:
For content I have hesitancy around optional dependencies, they have historically caused issues resulting in unstable lock files that result in dirty changes in unexpected conditions. May be FUD but just want to confirm the user benefit.

Reply:
Removed the `optional` for `@angular/localize`. My reasoning was just that clients don't necessarily need to use localization,  but since we expect SLE to, we can leave it off unless we have a need for it in the future.